ndscan/experiment/scan_runner.py

```diff
@@ -435,6 +438,9 @@ def _point_completed(self):
         # the axis coordinates.
         values = self._current_chunk.pop(0)
         for value, sink in zip(values, self._axis_sinks):
+            if isinstance(value, np.bool_):
+                # Convert as `np.bool_` is not PYON-serializable.
+                value = bool(value)
```

Comment:
What's going on here (and above)? Why are these `np.bool_`s in the first place?

Reply:
Probably because of `ListGenerator.points_for_level` (line 209).

Reply:
`LinearGenerator` returns `.tolist()` - should probably just do that in `ListGenerator`, too.